src/main/kotlin/snyk/container/ContainerService.kt

```diff
@@ -37,16 +37,17 @@ class ContainerService(project: Project) : CliAdapter<ContainerResult>(
         val images = imageCache?.getKubernetesWorkloadImages() ?: emptySet()
         val imageNames = images.map { it.image }
         LOG.debug("container scan requested for ${imageNames.size} images: $imageNames")
-
-        val result = execute(commands + imageNames)
-        if (!result.isSuccessful()) {
+        if (imageNames.isEmpty()) {
+            return ContainerResult(emptyList(), null)
+        }
+        val tempResult = execute(commands + imageNames)
+        if (!tempResult.isSuccessful()) {
             LOG.debug("container service scan fail")
-            return result
+            return tempResult
```

Comment:
Detekt - too many return statements...

Reply:
Hmm... Detekt IntelliJ plugin does not show it for me... 
Anyway it's just 3 clear returns here and that part of code will be changed to parallelStream implementation